src/current/v25.1/create-logically-replicated.md

```diff
@@ -0,0 +1,115 @@
+
+Once the offline initial scan completes, the new table will come online and is ready to serve queries. In a bidirectional setup, the second LDR stream will also initialize after the offline initial scan completes.
+
+{{site.data.alerts.callout_danger}}
+If the table to be replicated contains [user-defined types]({% link {{ page.version.version }}/enum.md %}) or [foreign key]({% link {{ page.version.version }}/foreign-key.md %}) dependencies, you must use the [`CREATE LOGICAL REPLICATION STREAM`]({% link {{ page.version.version }}/create-logical-replication-stream.md %}) statement instead. 
+{{site.data.alerts.end}}
+
+This page is a reference for the `CREATE LOGICALLY REPLICATED` SQL statement, which includes information on its parameters and options. For a step-by-step guide to set up LDR, refer to the [Set Up Logical Data Replication]({% link {{ page.version.version }}/set-up-logical-data-replication.md %}) page.
+
+## Required privileges
+
+`CREATE LOGICALLY REPLICATED` requires one of the following privileges:
+
+- The [`admin` role]({% link {{ page.version.version }}/security-reference/authorization.md %}#admin-role).
+- The [`REPLICATION` system privilege]({% link {{ page.version.version }}/security-reference/authorization.md %}#privileges).
+
+Use the [`GRANT SYSTEM`]({% link {{ page.version.version }}/grant.md %}) statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT SYSTEM REPLICATION TO user;
+~~~
+
+## Synopsis
+
+<div>
+{% include {{ page.version.version }}/ldr/create_logically_replicated_stmt.html %}
+</div>
+
+### Parameters
+
+Parameter | Description
+----------+------------
+`db_object_name` | The fully qualified name of the table on the source or destination cluster. Refer to [Examples](#examples).
+`logical_replication_resources_list` | A list of the fully qualified table names on the source or destination cluster to include in the LDR stream. Refer to the [LDR with multiple tables](#multiple-tables) example.
+`source_connection_string` | The connection string to the source cluster. Use an [external connection]({% link {{ page.version.version }}/create-external-connection.md %}) to store the source cluster's connection URI. To start LDR, you run `CREATE LOGICALLY REPLICATED` from the destination cluster.
+`logical_replication_create_table_options` | Options to modify the behavior of the LDR stream. For a list, refer to [Options](#options). **Note:** `bidirectional on` or `unidirectional`is a required option.
+
+## Options
+
+Option | Description
+-------+------------
+`bidirectional on` / `unidirectional` | (**Required**) Specifies whether the LDR stream will be unidirectional or bidirectional. With `bidirectional on` specified, LDR will set up two LDR streams between the clusters.
```

Comment:
same re: reference on bi/uni directional? i mean the words make sense about what they do but i'd be wondering about if there are caveats etc